### PR TITLE
Add PHPUnit tests for Helper/Cart 

### PR DESCRIPTION
**Pull request by YevhenShBolt**
Recreated from develop branch.

### CODE REVIEW

File: Test/Unit/Helper/CartTest.php [hunk 1]
```diff
@@ -0,0 +1,433 @@
...
+    }
+
+    /**
+     * @param bool $shouldBeEmpty
+     * @return array
+     */
+    private function getAddressData($shouldBeEmpty = false)
+    {
+        if ($shouldBeEmpty) {
+            $data = [
```
Comment (danac-gs) [inline Test/Unit/Helper/CartTest.php]: Please add doc string explaining shouldBeEmpty, it is confusing returning non empty data.

Comment (YevhenShBolt) [reply Test/Unit/Helper/CartTest.php]: Ah yeah, it is extra parameters. I will remove it.

File: Test/Unit/Helper/CartTest.php [hunk 2]
```diff
@@ -0,0 +1,410 @@
...
+                'street_address1' => "228 7th Avenue",
+                'street_address2' => null,
+                'locality' => "New York",
+                'region' => "New York",
+                'postal_code' => "10011",
+                'country_code' => "US",
+                'email'=> "integration@bolt.com"
+            ],
+            'discounts' => [],
+            'total_amount' => 10000.0,
```
Comment (daisy1754) [inline Test/Unit/Helper/CartTest.php]: this has to be integer. Bolt only accept int for amount field

File: Test/Unit/Helper/CartTest.php [hunk 3]
```diff
@@ -0,0 +1,410 @@
...
+        $this->quoteResource = $this->createMock(QuoteResource::class);
+        $this->sessionHelper = $this->createMock(SessionHelper::class);
+        $this->checkoutHelper = $this->createMock(CheckoutHelper::class);
+        $this->discountHelper = $this->createMock(DiscountHelper::class);
+    }
+
+    /**
+     * @test
+     */
+    public function getCartData()
```
Comment (daisy1754) [inline Test/Unit/Helper/CartTest.php]: let's be more descriptive. How about `getCartData_multistepNoDiscount`.

**Comment by YevhenShBolt** [comment]
I can not understand what is the error in the log - ci/circleci: unit-php72-magento23

**Comment by daisy1754** [comment]
@YevhenShBolt 

Here is error message:
https://circleci.com/gh/BoltApp/bolt-magento2/797?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

Here is PR to fix CI config so we can get error message
https://github.com/BoltApp/bolt-magento2/pull/216

**Comment by YevhenShBolt** [comment]
Fixed error for magento2.3 php7.2

**Pull request merged**